### ISSUE TITLE
Callbacks always return `null`

### ISSUE THREAD

**Issue by TomSeldon** [ionic-team/cordova-plugin-ios-keychain#4]
I've tried using this plugin as described in `README.md` but I can't seem to get it to work. There are no errors, but on setting and getting a key/value the success callback is always called with `null` as the only argument.

Am I doing something wrong?

See below output which was run on an iPad running iOS9:

![screen shot 2016-08-11 at 09 33 32](https://cloud.githubusercontent.com/assets/3420620/17583035/e414d750-5fa6-11e6-9173-1e2d6093e4c0.png)

I'd expect the final call, `Keychain.get(onSuccess, onFail, 'someKey', 'message')` to call the success callback with `value` (the string set in the previous call). This doesn't happen.

**Comment by tmct** [comment]
This might be caused by #5 - on our devices the Objective-C was never successfully setting the value in the keychain because it was trying to use TouchID

**Comment by Gilgamoosh** [comment]
I am also having this issue, and I noticed something else that is weird as well. 
The success function (argument 1 of Keychain.get) takes a 'value'. If I check that value in the function, for example:
`function (value) { alert(value); }`
Then in the alert I get that the value is true/false based on what I set the key's value to be. But if I try to assign that value in the function, like so:

```
var example;
Keychain.get( function (value) { example = value; alert(example); }, . . .[other arguments]. . .);
alert(example);
```

then I find that the alert inside Keychain.get shows example having a value equal to the key's value, but the alert outside Keychain.get shows example having no value.

This seems related so I'm noting it here instead of creating another issue, but I could be completely wrong on that.

**Comment by tmct** [comment]
That's just because your third line (the `alert`) is not guaranteed to be called after the asynchronous callback that sets `example`'s value :)

**Comment by Gilgamoosh** [comment]
Ah, I didn't catch that. Thank you. I've applied the change from #5 so I'll see how that goes.
Update for information: The change that fixes #5 can be found in #8 

UPDATE: The change for #5 did not fix the issue.

**Comment by Gilgamoosh** [comment]
Alright more information:
I've ran through a set and a get in xcode debugger. Context: I confirmed that the initial value was false. I then set it to true, and used get to determine if the value had changed in the keychain.

![vnc-viewer-5 3 2-windows-64bit_2016-08-17_13-00-56](https://cloud.githubusercontent.com/assets/12585467/17745758/15faa87a-647b-11e6-8110-644ff1fa1453.png)
Value was false (the initial get not shown to conserve space). Now value is true by the end of the set.

![vnc-viewer-5 3 2-windows-64bit_2016-08-17_13-02-45](https://cloud.githubusercontent.com/assets/12585467/17745799/42e61270-647b-11e6-88ab-2886853c0395.png)
When I use get, it shows the value as true, so it is definitely accessing the keychain and storing a key-value pair there. 

I'm not very well versed in Objective-C, and I also did not commit a lot of time to understanding this (yet), but might the issue be related to the get method returning (void) instead of (NSData *) or (NSString *)?
